python/sglang/srt/server_args.py

```diff
@@ -928,7 +928,10 @@ def init_new(server_args) -> "PortArgs":
         while True:
             if is_port_available(port):
                 break
-            port += 42
+            if port < 60000:
+                port += random.randint(1, 100)
```

Comment:
Do not only add 1. Add at least 42, so we can launch many engines together without port conflicts